src/marketplace-custom-config.js

```diff
@@ -26,8 +26,26 @@ export const priceFilterConfig = {
 };
 
 // Activate booking dates filter on search page
-export const dateRangeFilterConfig = {
+export const dateRangeLengthFilterConfig = {
   active: true,
+
+  // A global time zone to use in availability searches. As listings
+  // can be in various time zones, we must decide what time zone we
+  // use in search when looking for available listings within a
+  // certain time interval.
+  //
+  // If you have all/most listings in a certain time zone, change this
+  // config value to that.
+  //
+  // See: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+  searchTimeZoneId: 'Etc/UTC',
```

Comment:
We should probably mention something about this in 
https://www.sharetribe.com/docs/cookbook-search/change-search-filters-in-ftw/#filter-types

Reply:
Sure, I have another story for doc updates. I added that link to the story.